pkg/gw/safeset_handler_test.go

```diff
@@ -0,0 +1,115 @@
+package gw
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	immuclient "github.com/codenotary/immudb/pkg/client"
+	"github.com/codenotary/immudb/pkg/server"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/lithammer/shortuuid"
+)
+
+func TestSafeset(t *testing.T) {
+	op := server.DefaultOptions()
+	op.Dir = "db_" + shortuuid.New()
```

Comment:
Is it worth using a 3rd party library for this? I am thinking maybe just using the current unix timestamp would be good enough.
Usually we try to avoid adding a new dependency, unless it is really worth it.

Reply:
Refactored the test and removed the dependency